### PR TITLE
Fixed styles for generated names on desktop

### PR DESCRIPTION
**Pull request by flexsurfer**
Fixed styles for generated names on desktop

image

### CODE REVIEW

**Comment by yenda** [comment]
This will conflict with https://github.com/status-im/status-react/pull/6652 I suggest you rebase on it. Doing things the way #6652 does will prevent someone abusing rtl chars

**Comment by flexsurfer** [comment]
@EugeOrtiz could you please take a look at the screenshot in the description? tbh I like it more when all 3-w names are grey, do no distract from messages @errorists what do you think also?

**Comment by flexsurfer** [comment]
image

**Review by rcullito — APPROVED** [review]
nice :+1: seems to increase readability imho

**Comment by errorists** [comment]
@flexsurfer 👏 Huge improvement! Ideally, we should use the medium font weight for usernames as bold is a bit fat, font-weight: 500 should do the trick.

**Comment by flexsurfer** [comment]
@errorists agree, bold looks fat on desktop

**Review by vitvly — APPROVED** [review]

**Comment by EugeOrtiz** [comment]
@flexsurfer @errorists  looking good! But the name is not bold, here are the design specs:

**Username**: Semibold 14 (weight 600) /  Line-height: 16 / Black
**Anonymous name**: Regular 14 (weight 400) / Line-height: 16 / Dark grey
**Dot**: Dark grey
**Paragraph/Message**: Regular 14 (weight 400) / Line-height: 20 / Black
**Timestamp**: Regular 10 (weight 400) /  Line-height: 12 / Dark Grey

[Designs on Figma](https://www.figma.com/file/MRYBI5axcTwL4Ft8xAYhFOfy/Desktop-Chat?node-id=134%3A1887) (select an element to see the specs)

Screenshot:
captura de pantalla 2018-11-26 a la s 9 08 30 a m

**Comment by SlizkovaGanna** [comment]
Tested on Mac, Linux (emulated) - looks good.
1 of the fails I could not reproduce, the other is expected.

**Perhaps we should also consider changing the font in the reply bubble?**
For example, on a screenshot below the first quote is from a user added to contacts (the first name is displayed in bold font in chats), and the second quote is from a user not in contacts (name is displayed in grey)
@errorists @flexsurfer Maybe we should consider making the name in quote bubble also bold for future implementation?

image

**Comment by EugeOrtiz** [comment]
+1 @SlizkovaGanna should we create a new issue or @flexsurfer can you apply it on this issue?
I added the specs to [Figma](https://www.figma.com/file/MRYBI5axcTwL4Ft8xAYhFOfy/Desktop-Chat?node-id=134%3A1887):

captura de pantalla 2018-11-26 a la s 10 29 12 a m

**Comment by flexsurfer** [comment]
Bold
image



600
image




500
image

**Comment by flexsurfer** [comment]
@EugeOrtiz @SlizkovaGanna  i changed from bold to 600, lets create new issues for quotes

File: src/status_im/ui/screens/desktop/main/chat/styles.cljs [hunk 1]
```diff
@@ -37,9 +37,13 @@
    :width              230})
 
 (def author
-  {:color         colors/black
-   :font-size     14
-   :font-weight   :bold})
+  {:color       colors/black
+   :font-weight 600
```
Comment (yenda) [inline src/status_im/ui/screens/desktop/main/chat/styles.cljs]: font-weight has to be a string

Comment (flexsurfer) [reply src/status_im/ui/screens/desktop/main/chat/styles.cljs]: good catch, interesting why there is a difference between 600 and 500 then

Comment (yenda) [reply src/status_im/ui/screens/desktop/main/chat/styles.cljs]: Maybe it's only a problem on Linux

Comment (errorists) [reply src/status_im/ui/screens/desktop/main/chat/styles.cljs]: With font weight 400 is regular 500 is medium, 600 semi bold and 700 bold :)

**Comment by flexsurfer** [comment]
@EugeOrtiz @SlizkovaGanna so semibold doesn't work on desktop, so i used medium, let's merge this PR because it took too much time for this change because current bold names look awful

**Pull request merged**